sql/sql_plugin.cc

```diff
@@ -91,6 +91,9 @@ const LEX_CSTRING plugin_type_names[MYSQL_MAX_PLUGIN_TYPE_NUM]=
   { STRING_WITH_LEN("AUTHENTICATION") },
   { STRING_WITH_LEN("PASSWORD VALIDATION") },
   { STRING_WITH_LEN("ENCRYPTION") }
+#ifdef WITH_WSREP
+  , { STRING_WITH_LEN("WSREP") }
+#endif
```

Comment:
If we decide to stay with the new plugin type approach, it should not be under ifdefs. We can't reuse this plugin number for anything else anyway.